requirements.txt

```diff
@@ -0,0 +1,3 @@
+argparse
+psutil
+toml>=0.9.0
```

Comment:
On my side, the packages needed seem to be:
* colorama
* psutil
* toml
* pillow

```
afontain@thinkpad-nixos ~/unv$ Urcheon/bin/urcheon                             
/usr/bin/env: ‘python3’: No such file or directory
afontain@thinkpad-nixos ~/unv$ nix-shell -p python38 --run Urcheon/bin/urcheon  Traceback (most recent call last):
  File "Urcheon/bin/urcheon", line 18, in <module>
    from Urcheon import Urcheon
  File "/home/antoine/unv/Urcheon/Urcheon/Urcheon.py", line 12, in <module>
    from Urcheon import Pak
  File "/home/antoine/unv/Urcheon/Urcheon/Pak.py", line 11, in <module>
    from Urcheon import Action
  File "/home/antoine/unv/Urcheon/Urcheon/Action.py", line 11, in <module>
    from Urcheon import FileSystem
  File "/home/antoine/unv/Urcheon/Urcheon/FileSystem.py", line 11, in <module>
    from Urcheon import Ui
  File "/home/antoine/unv/Urcheon/Urcheon/Ui.py", line 12, in <module>
    from colorama import Fore, Style, init
ModuleNotFoundError: No module named 'colorama'

[…]

afontain@thinkpad-nixos ~/unv$ nix-shell -p python38 python38Packages.{colorama,psutil,toml,pillow}  --run Urcheon/bin/urcheon
usage: urcheon [-h] <stage> [stage arguments]
```

* colorama comes from Ui.py
* PIL (pillow) comes from Bsp.py

Reply:
ah yes I forgot them